README.md

```diff
@@ -52,17 +67,32 @@ This command would need customizing for your environment.  [This link](https://s
 
 Once you have a certificate and key file name them `stash.crt` and `stash.key` and place them in the `~/.stash` directory.  Stash detects these and starts up using HTTPS rather than HTTP.
 
+# Customization
+
+You can make Stash interface fit your desired style with  [Custom CSS snippets](https://github.com/stashapp/stash/wiki/Custom-CSS-snippets) and [CSS Tweaks](https://github.com/stashapp/stash/wiki/CSS-Tweaks).
+
+-[Plex](https://github.com/stashapp/stash/wiki/Stash-Plex-Theme) is a community created theme inspired by popular Plex Interface.
+[Pulsar](https://github.com/stashapp/stash/wiki/Pulsar-Theme) is another option.
```

Comment:
I think we're better off making a generic `Themes` wiki page (with links to theme pages) and linking to that, rather than linking individual themes.

Reply:
Should I leave this version for now and adjust when that page is created?